Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -249,7 +273,7 @@ extension PlaceCarouselViewController: CLLocationManagerDelegate {
         mapView.region = MKCoordinateRegion(center: center, span: span)
 
         FirebasePlacesDatabase().getPlaces(forLocation: location).upon(DispatchQueue.main) { places in
-            self.places = places.flatMap { $0.successResult() }
+            self.places = PlaceUtilities.sort(places: places.flatMap { $0.successResult() }, byDistanceFromLocation: location)
```

Comment:
I'm not sure we want to sort *all* places by absolute distance but works for now.